website/docs/api/spancategorizer.md

```diff
@@ -13,6 +13,22 @@ A span categorizer consists of two parts: a [suggester function](#suggesters)
 that proposes candidate spans, which may or may not overlap, and a labeler model
 that predicts zero or more labels for each candidate.
 
+Predicted spans will be saved in a [`SpanGroup`](/api/spangroup) on the doc.
+Individual span scores can be found in `spangroup.attrs["scores"]`.
+
+## Annotation Format {#annotations}
```

Comment:
I think "Annotation Format" is a confusing name for this. In the factory it's called "assigns", which seems less confusing with training corpus annotation formats, but I'm still not sure what a good section name is here...

Reply:
I agree - to me "annotation" refers to the input format of the training data, while this section describes the resulting output (predictions) of the component. Hm. Something like "Predicted fields"? "Output format"? :|

Reply:
I'm not sure "annotation" is the wrong word here - when components set values during training, don't we call them "annotating components"? I think this is also in line with the v3 design that an Example is just a manually created version of the hoped-for output; the format is what the user will need to provide in training anyway. 

That said "Predicted fields" or "Assigned fields" also sound fine. "Output format" also words but I guess I avoided that because it's a side effect.

Reply:
Ah, that's actually a good point, as the `Example` objects take their annotated format on the `Docs`, the input and output format is actually the same!

Reply:
It still reads way to much like "corpus annotation format" for me, which I wouldn't associate with a `Doc` even though I understand that it's technically equivalent. I feel like "predicted", "field", and "format" are a bit vague or confusing.

Maybe "Assigned Attributes"? We also say "attributes" here: https://spacy.io/api/language#analyze_pipes

Reply:
I like "Assigned Attributes"! Should we go with that?

Reply:
Fine by me!

Reply:
With that resolved I think this is good to go?